app/src/main/java/com/devindi/wallpaper/settings/model/StringField.kt

```diff
@@ -0,0 +1,18 @@
+package com.devindi.wallpaper.settings.model
+
+import android.content.SharedPreferences
+
+class StringField(private val storage: SharedPreferences, override val key: String) : SettingsField<String> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/f22b9ee68b14344acb1bd6cd546c2767866b4eef/app/src/main/java/com/devindi/wallpaper/settings/model/StringField.kt#L5"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>